src/Entity/Attachment/Attachment.php

```diff
@@ -120,11 +120,16 @@ public function getDescription()
      * Returns the url to a specific picture config (size, branding, etc...)
      *
      * @param string $config
+     * @param bool   $useBrandedImage
      *
      * @return string
      */
-    public function getUrlForConfig($config = 'orig')
+    public function getUrlForConfig($config = 'orig', $useBrandedImage = true)
     {
+        if ($useBrandedImage === false) {
```

Comment:
Not every config has an unbranded version, and this would break image urls of configs without an unbranded version.
https://api-docs.justimmo.at/reference/picture-sizes.html

In addition this is something which should not be done in the sdk but rather in the code using it. The outer code already determines the config to be displayed and if its branded or unbranded so he can change the config by himself and just pass in the correct final config. The sdk does not need to know about branded / unbranded.